src/autoload.php

```diff
@@ -7,7 +7,17 @@
  */
 
 $autoloadFiles = [
+    // Attempt to translate the /vendor/ in the directory path to the vendor directory.
     preg_replace('#/vendor/.*$#', '/vendor/wordpress-autoload.php', __DIR__),
+    preg_replace('/\\\vendor\\\.*$/', '/vendor/wordpress-autoload.php', __DIR__),
```

Comment:
This looks like one too many backslashes - shouldn't it be like this?
```suggestion
    preg_replace('/\\vendor\\.*$/', '/vendor/wordpress-autoload.php', __DIR__),
```

Reply:
I'm not a regex pro to know but in testing with `C:\Users\Sean Fisher\Local Sites\wordpress\app\public\wp-content\vendor\alleyinteractive\composer-wordpress-autoloader\src` as the `__DIR__`:

four slashes: `preg_replace('/\\\vendor\\\.*$/', '/vendor/wordpress-autoload.php', __DIR__),` -> `"C:\Users\Sean Fisher\Local Sites\wordpress\app\public\wp-content/vendor/wordpress-autoload.php`

two slashes: `preg_replace('/\\vendor\\\.*$/', '/vendor/wordpress-autoload.php', __DIR__),` -> `C:\Users\Sean Fisher\Local Sites\wordpress\app\public\wp-content\vendor\alleyinteractive\composer-wordpress-autoloader\src`

https://onlinephp.io?s=lc8_C8IwEAXw2UK_QwYhKtbu_sFBcHIrbgdyJocGYnNc0ha_vRHRRRfXx-P3eOstX7ksxtaJ2ii9W8IxkkRoCFu1d_FKAodg0KvGJYowBLEsFCMgM3B39s7AwJUJbaI2QU-tDQLoPd1djgRNcj2BCTcOWa4-QIVdCj6gzQtRjF6VRVn0KCfb3XhSFqPcupyE2KOhia4B3jjAYjau9Vzp-pXU3-gi38qN56_p_Bf2tv6mpqsH&v=8.2.13

/shrug